src/simulate.py

```diff
@@ -507,18 +567,34 @@ def main(args):
         description="Runs a hardware simulation on a given benchmark and technology spec",
         epilog="Text at the bottom of help",
     )
+
+    # eventually 
+    parser.add_argument( 
+        "--parasitics",
+        type=str,
+        choices=["detailed", "estimation", "none"],
+        default="none",
+        help="determines what type of parasitic calculations are done for wires",
+    )
+
+    parser.add_argument(
+        "--openroad_testfile",
+        type=str,
+        default="openroad_interface/tcl/test_nangate45_bigger.tcl",
+        help="what tcl file will be executed for openroad",
+    )
```

Comment:
this determines what .lef files to look at right?

Reply:
yeah basically